src/main/cms/cms_menu_ledstrip.c

```diff
@@ -68,10 +95,84 @@ static long cmsx_Ledstrip_FeatureWriteback(const OSD_Entry *self)
     return 0;
 }
 
+// Names for colorId_e. Zero (black position) is used for disabled.
+static const char * const cmsx_Ledstrip_ColorNames[] = {
```

Comment:
I'd rather go for HSV or RGB values by numbers than for hardcoded colours - this will require a firmware update whenever a race organiser specifies a colour not on this list.